sys/mips/gt64120.c

```diff
@@ -396,13 +396,18 @@ static device_t *gt_pci_identify(driver_t *drv, device_t *parent) {
   return device_add_child(parent, &DEVCLASS(pci), 0);
 }
 
+static int gt64120_probe(device_t *d) {
+  return 1;
+}
+
 /* clang-format off */
 pci_bus_driver_t gt_pci_bus = {
   .driver = {
     .desc = "GT-64120 PCI bus driver",
     .size = sizeof(gt_pci_state_t),
     .attach = gt_pci_attach,
     .identify = gt_pci_identify,
+    .probe = gt64120_probe
```

Comment:
Why `gt64120_probe`? All other functions here are `gt_pci_*`.

Reply:
Changed to gt_pci_probe.